Babylon/commands/api/payloads/upsert.py

```diff
@@ -0,0 +1,127 @@
+import jmespath
+import yaml
+import click
+import logging
+import pathlib
+
+from typing import Any
+from click import Path, argument, command, option
+from Babylon.utils.environment import Environment
+from Babylon.utils.response import CommandResponse
+from flatten_json import flatten, unflatten_list
+from ruamel.yaml import YAML
+
+from Babylon.utils.update_section import get_section_and_replace
+
+logger = logging.getLogger("Babylon")
+env = Environment()
+
+
+@command()
+@option("--flatten", "flat", is_flag=True, help="Flatten option")
+@option("--dashboard_view", "dashboard_view", is_flag=True, help="dashboard_view section")
+@option("--scenario_view", "scenario_view", is_flag=True, help="scenario_view section")
+@option("--origin", "origin_file", required=True, type=Path(path_type=pathlib.Path), help="yaml file to change")
+@option("--target", "target_file", required=True, type=Path(path_type=pathlib.Path), help="yaml file to dump changes")
+@option("--query", "query", type=str, help="Jmespath query")
+@option("--section", "section", type=str, help="section in yaml file to perform changes")
+@argument("stream")
+def upsert(origin_file: pathlib.Path, target_file: pathlib.Path, query: str, section: str, flat: bool, stream: Any,
```

Comment:
i need this to wrap the stdin because upsert expects an argument
```text
line 37 data_s = click.get_text_stream('stdin', encoding="utf-8")
```

 for example:
```bash 
# update one key
babylon -prj supply -plt perf api payloads upsert \
    --origin .payload/supply.perf.solution.update.yaml \
    --target .payload/supply.perf.solution.update.yaml \
    --query "parameterGroups[?id=='simulation'] | {\"key\":'parameterGroups', value:[0]}" -<<EOF
- id: simulation
  labels:
    en: Simulation EN
    fr: Simulation FR
  parameters:
  - scenario_name
  - start_date
  - end_date
  - simulation_granularity
EOF
```

Reply:
Understood. So maybe a 'help' section inside the definition of 
```python
@argument("stream")
``` 
?